builder/dockerfile/dispatchers.go

```diff
@@ -211,7 +211,7 @@ func from(b *Builder, args []string, attributes map[string]bool, original string
 			return errors.Errorf("invalid name for build stage: %q, name can't start with a number or contain symbols", ctxName)
 		}
 	} else if len(args) != 1 {
-		return errExactlyOneArgument("FROM")
+		return errors.New("FROM requires exactly one or three arguments")
```

Comment:
Perhaps _either_ one or three arguments

```
FROM requires either one or three arguments
```

Perhaps (as a follow up) we should enhance the error message in the CLI, and provide some concrete examples (`FROM foo:bar` or `FROM foo:bar AS baz`) - just thinking out loud here